dvc/remote/local.py

```diff
@@ -335,8 +341,14 @@ def _save_file(self, path_info):
     def _save_dir(self, path_info):
         path = path_info['path']
         md5, dir_info = self.state.update_info(path)
+        dir_relpath = os.path.relpath(path)
+        dir_size = len(dir_info)
+        bar = dir_size > LARGE_DIR_SIZE
```

Comment:
I would also propagate an actual verbose mode here. If `--verbose` is specified don't show bar, show individual entry.

Reply:
That's a good suggestion, @shcheklein. However, I think there's no easy way to access the arguments from the `RemoteLOCAL` :disappointed: .

Maybe that implementation could be on `process`, or on `logger`, not logging with the progress bar if the `--verbose` output is specified.